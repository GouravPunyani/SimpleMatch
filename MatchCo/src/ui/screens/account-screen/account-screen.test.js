import 'react-native'
import React from 'react'
import renderer from 'react-test-renderer'
import Screen from './account-screen'

test('renders correctly', () => {
  const tree = renderer.create(<Screen />)
  expect(tree).toMatchSnapshot()
})
