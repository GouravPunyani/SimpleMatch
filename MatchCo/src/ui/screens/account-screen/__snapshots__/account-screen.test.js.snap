// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<RCTScrollView
  contentContainerStyle={
    Object {
      "alignItems": "center",
    }
  }
  style={
    Object {
      "backgroundColor": "rebeccapurple",
      "flex": 1,
    }
  }
>
  <View>
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      style={
        Object {
          "color": "white",
          "fontSize": 30,
        }
      }
    >
      AccountScreen
    </Text>
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      style={
        Object {
          "color": "white",
          "fontSize": 12,
        }
      }
    >
      src/ui/screens/account-screen/account-screen.js
    </Text>
    <View
      style={
        Object {
          "alignItems": "center",
          "justifyContent": "center",
          "margin": 10,
        }
      }
    >
      <View
        accessibilityComponentType={undefined}
        accessibilityLabel={undefined}
        accessibilityTraits={undefined}
        accessible={true}
        hitSlop={undefined}
        isTVSelectable={true}
        onLayout={undefined}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          Object {
            "alignItems": "center",
            "borderColor": "white",
            "borderRadius": 4,
            "borderWidth": 1,
            "justifyContent": "center",
            "margin": 4,
            "minHeight": 44,
            "minWidth": 44,
            "opacity": 1,
            "paddingHorizontal": 10,
            "paddingVertical": 4,
          }
        }
        testID={undefined}
        tvParallaxProperties={undefined}
      >
        <Text
          accessible={true}
          allowFontScaling={true}
          ellipsizeMode="tail"
          style={
            Array [
              Object {
                "color": "#333333",
                "fontFamily": "Nexa",
                "fontSize": 14,
              },
            ]
          }
        >
          Current Orders
        </Text>
      </View>
      <View
        accessibilityComponentType={undefined}
        accessibilityLabel={undefined}
        accessibilityTraits={undefined}
        accessible={true}
        hitSlop={undefined}
        isTVSelectable={true}
        onLayout={undefined}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          Object {
            "alignItems": "center",
            "borderColor": "white",
            "borderRadius": 4,
            "borderWidth": 1,
            "justifyContent": "center",
            "margin": 4,
            "minHeight": 44,
            "minWidth": 44,
            "opacity": 1,
            "paddingHorizontal": 10,
            "paddingVertical": 4,
          }
        }
        testID={undefined}
        tvParallaxProperties={undefined}
      >
        <Text
          accessible={true}
          allowFontScaling={true}
          ellipsizeMode="tail"
          style={
            Array [
              Object {
                "color": "#333333",
                "fontFamily": "Nexa",
                "fontSize": 14,
              },
            ]
          }
        >
          Completed Orders
        </Text>
      </View>
      <View
        accessibilityComponentType={undefined}
        accessibilityLabel={undefined}
        accessibilityTraits={undefined}
        accessible={true}
        hitSlop={undefined}
        isTVSelectable={true}
        onLayout={undefined}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          Object {
            "alignItems": "center",
            "borderColor": "white",
            "borderRadius": 4,
            "borderWidth": 1,
            "justifyContent": "center",
            "margin": 4,
            "minHeight": 44,
            "minWidth": 44,
            "opacity": 1,
            "paddingHorizontal": 10,
            "paddingVertical": 4,
          }
        }
        testID={undefined}
        tvParallaxProperties={undefined}
      >
        <Text
          accessible={true}
          allowFontScaling={true}
          ellipsizeMode="tail"
          style={
            Array [
              Object {
                "color": "#333333",
                "fontFamily": "Nexa",
                "fontSize": 14,
              },
            ]
          }
        >
          Personal Info
        </Text>
      </View>
      <View
        accessibilityComponentType={undefined}
        accessibilityLabel={undefined}
        accessibilityTraits={undefined}
        accessible={true}
        hitSlop={undefined}
        isTVSelectable={true}
        onLayout={undefined}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          Object {
            "alignItems": "center",
            "borderColor": "white",
            "borderRadius": 4,
            "borderWidth": 1,
            "justifyContent": "center",
            "margin": 4,
            "minHeight": 44,
            "minWidth": 44,
            "opacity": 1,
            "paddingHorizontal": 10,
            "paddingVertical": 4,
          }
        }
        testID={undefined}
        tvParallaxProperties={undefined}
      >
        <Text
          accessible={true}
          allowFontScaling={true}
          ellipsizeMode="tail"
          style={
            Array [
              Object {
                "color": "#333333",
                "fontFamily": "Nexa",
                "fontSize": 14,
              },
            ]
          }
        >
          Payment Methods
        </Text>
      </View>
      <View
        accessibilityComponentType={undefined}
        accessibilityLabel={undefined}
        accessibilityTraits={undefined}
        accessible={true}
        hitSlop={undefined}
        isTVSelectable={true}
        onLayout={undefined}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          Object {
            "alignItems": "center",
            "borderColor": "white",
            "borderRadius": 4,
            "borderWidth": 1,
            "justifyContent": "center",
            "margin": 4,
            "minHeight": 44,
            "minWidth": 44,
            "opacity": 1,
            "paddingHorizontal": 10,
            "paddingVertical": 4,
          }
        }
        testID={undefined}
        tvParallaxProperties={undefined}
      >
        <Text
          accessible={true}
          allowFontScaling={true}
          ellipsizeMode="tail"
          style={
            Array [
              Object {
                "color": "#333333",
                "fontFamily": "Nexa",
                "fontSize": 14,
              },
            ]
          }
        >
          Shipping Address
        </Text>
      </View>
      <View
        accessibilityComponentType={undefined}
        accessibilityLabel={undefined}
        accessibilityTraits={undefined}
        accessible={true}
        hitSlop={undefined}
        isTVSelectable={true}
        onLayout={undefined}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          Object {
            "alignItems": "center",
            "borderColor": "white",
            "borderRadius": 4,
            "borderWidth": 1,
            "justifyContent": "center",
            "margin": 4,
            "minHeight": 44,
            "minWidth": 44,
            "opacity": 1,
            "paddingHorizontal": 10,
            "paddingVertical": 4,
          }
        }
        testID={undefined}
        tvParallaxProperties={undefined}
      >
        <Text
          accessible={true}
          allowFontScaling={true}
          ellipsizeMode="tail"
          style={
            Array [
              Object {
                "color": "#333333",
                "fontFamily": "Nexa",
                "fontSize": 14,
              },
            ]
          }
        >
          Contact Us
        </Text>
      </View>
      <View
        accessibilityComponentType={undefined}
        accessibilityLabel={undefined}
        accessibilityTraits={undefined}
        accessible={true}
        hitSlop={undefined}
        isTVSelectable={true}
        onLayout={undefined}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          Object {
            "alignItems": "center",
            "borderColor": "white",
            "borderRadius": 4,
            "borderWidth": 1,
            "justifyContent": "center",
            "margin": 4,
            "minHeight": 44,
            "minWidth": 44,
            "opacity": 1,
            "paddingHorizontal": 10,
            "paddingVertical": 4,
          }
        }
        testID={undefined}
        tvParallaxProperties={undefined}
      >
        <Text
          accessible={true}
          allowFontScaling={true}
          ellipsizeMode="tail"
          style={
            Array [
              Object {
                "color": "#333333",
                "fontFamily": "Nexa",
                "fontSize": 14,
              },
            ]
          }
        >
          FAQs
        </Text>
      </View>
      <View
        accessibilityComponentType={undefined}
        accessibilityLabel={undefined}
        accessibilityTraits={undefined}
        accessible={true}
        hitSlop={undefined}
        isTVSelectable={true}
        onLayout={undefined}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          Object {
            "alignItems": "center",
            "borderColor": "white",
            "borderRadius": 4,
            "borderWidth": 1,
            "justifyContent": "center",
            "margin": 4,
            "minHeight": 44,
            "minWidth": 44,
            "opacity": 1,
            "paddingHorizontal": 10,
            "paddingVertical": 4,
          }
        }
        testID={undefined}
        tvParallaxProperties={undefined}
      >
        <Text
          accessible={true}
          allowFontScaling={true}
          ellipsizeMode="tail"
          style={
            Array [
              Object {
                "color": "#333333",
                "fontFamily": "Nexa",
                "fontSize": 14,
              },
            ]
          }
        >
          Privacy Policy
        </Text>
      </View>
      <View
        accessibilityComponentType={undefined}
        accessibilityLabel={undefined}
        accessibilityTraits={undefined}
        accessible={true}
        hitSlop={undefined}
        isTVSelectable={true}
        onLayout={undefined}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          Object {
            "alignItems": "center",
            "borderColor": "white",
            "borderRadius": 4,
            "borderWidth": 1,
            "justifyContent": "center",
            "margin": 4,
            "minHeight": 44,
            "minWidth": 44,
            "opacity": 1,
            "paddingHorizontal": 10,
            "paddingVertical": 4,
          }
        }
        testID={undefined}
        tvParallaxProperties={undefined}
      >
        <Text
          accessible={true}
          allowFontScaling={true}
          ellipsizeMode="tail"
          style={
            Array [
              Object {
                "color": "#333333",
                "fontFamily": "Nexa",
                "fontSize": 14,
              },
            ]
          }
        >
          Terms of Use
        </Text>
      </View>
    </View>
  </View>
</RCTScrollView>
`;
